### DISCUSSION TITLE
Video Tutorials

### DISCUSSION THREAD

**Discussion by eileenrmartin** [DASDAE/dascore#75] [Show and tell]
Following up on our last DASDAE team check-in meeting, I made the first of a series of at least 4 videos to help folks find out about resources and walk through the development process step-by-step. This 1st video goes over basic resources, and how to do a development install: [https://youtu.be/3tY1MijEbc4](https://youtu.be/3tY1MijEbc4)

**Comment by d-chambers** [comment]
This is awesome @eileenrmartin, I loved the environment solving music! We should see if Anaconda inc. will build the music and cat pictures into their command line tool.

**Comment by eileenrmartin** [comment]
2nd video in the series on testing basics, working with branches in this context (be sure to get comfortable with git first via resources in 1st video) and what it looks like to run pre-commit hooks: [https://youtu.be/eoxTiFga4VA](https://youtu.be/eoxTiFga4VA)

Nothing required long waits, so no cats or music this time, but maybe in another one!
